README.md

```diff
@@ -1 +1,217 @@
+    </li>
+     <!-- Resources -->
+    <li>
+      <a href="#resources">Resources</a>
+    </li>
+</ol>
+</details> <!-- END OF TABLE OF CONTENTS DROP DOWN -->
+
+## About the App
+
+* What does the app do & why did we build it?
+
+
+## Installation
+
+1. Clone the repo
+```
+git clone https://github.com/rpp30-boc-heartbleed/whimsical.git
+```
+2. Install NPM packages from the root directory and from the client directory.
+```
+npm install
+```
+
+## Getting Started
+
+1. Start the server from the root directory and confirm connection to database.
+```
+npm start
+```
+2. Start the the application from the client directory.
+```
+expo start
+```
+
+3. To test run:
+```
+npm test
+```
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Tech Stack
+
+### Frontend
+<a href='https://reactnative.dev/'>
+<img src="https://raw.githubusercontent.com/kristerkari/react-native-svg-transformer/HEAD/images/react-native-logo.png" width="60" height="70">
+</a>
+
+
+<a href='https://recoiljs.org/'>
+<img src="https://recoiljs.org/img/logo.svg"
+width="70" height="70">
+</a>
+
+
+### Backend
+
+<a href='https://expressjs.com/'><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/64/Expressjs.png/330px-Expressjs.png" width="70" height="25">
+</a>
+<a href="https://nodejs.org/en/"><img src="https://upload.wikimedia.org/wikipedia/commons/d/d9/Node.js_logo.svg" width="70" height="30">
+ </a>
+
+
+### Database
+<a href='https://www.mongodb.com/'><img src="https://cacm.acm.org/system/assets/0002/7119/042117_Theodo_MongoDB.large.jpg?1492791427&1492791427" width="70" height="70">
+</a>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+## How it Works
+
+<!-- Insert what app looks like visually, diagrams, demo video-->
+
+### API
+|Request Type|Endpoint                          |Status |
+|:----------:|----------------------------------|:-----:|
+|    GET     | /qb/something                    |200    |
+|    GET     | /qb/something                    |200    |
+|    POST    | /qb/something                    |201    |
+|    POST    | /qb/something                    |201    |
+|    PUT     | /qb/something                    |204    |
+|    PUT     | /qb/something                    |204    |
+|    PUT     | /qb/something                    |204    |
+|    PUT     | /qb/something                    |204    |
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Challenges
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Research
+<a href="https://trello.com/b/yvrqtpK9/rpp30-boc-heartbleed">
+Project Board
+</a>
+
+### User Stories
+<a href="https://docs.google.com/spreadsheets/d/1heVGbScj-MP8areUMJkZk7aNSUPD_GR8Hwmd6uMnrQ8/edit#gid=484004821">
+View stories >>
+</a>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Features
+- [ ] Placeholder Feature
+- [ ] Placeholder Feature
+- [x] Feature Implemented
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Team
```

Comment:
We can try to change the square images into circles if we decide to keep the images in the Team section - in the near future.